src/lib.rs

```diff
@@ -139,6 +139,17 @@ pub(crate) const CRINGE_ESLINT_DISABLE: &str = "/* eslint-disable */
 // #[cfg(doctest)]
 // doc_comment::doctest!("../README.md");
 
+/// The configuration for the generator
+#[derive(Default)]
+pub struct ExportConfiguration {
```

Comment:
This type seems impossible to use right now given it has private fields and no methods to set them publically other than the `Default` implementation. Having the fields private fields is the right call so the best way forward is a builder pattern (similar to how `specta::ts::ExportConfiguration` already is.

We should add the methods to `ExportConfiguration`:
 - `pub fn new(name: ExportConfiguration) -> Self`
 - `pub fn plugin_name(name: impl AsRef<Cow<'static, str>>) -> Self`

Then we should also add an implementation for `impl From<specta::ts::ExportConfiguration> for ExportConfiguration` (or `Into` if `From` doesn't work).